consent-ws/src/main/java/org/broadinstitute/consent/http/resources/ConsentsResource.java

```diff
@@ -1,15 +1,24 @@
 package org.broadinstitute.consent.http.resources;
 
 import com.google.common.base.Optional;
+import io.dropwizard.auth.Auth;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.log4j.Logger;
 import org.broadinstitute.consent.http.models.Consent;
+import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.models.dto.ConsentGroupNameDTO;
 import org.broadinstitute.consent.http.service.AbstractConsentAPI;
 import org.broadinstitute.consent.http.service.ConsentAPI;
 
 import javax.annotation.security.PermitAll;
+import javax.annotation.security.RolesAllowed;
 import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
 import java.util.*;
+import java.util.List;
```

Comment:
you can remove this import since you have java.util.*

Reply:
This was a change made by the IDE. I was thinking to put as it was, what is the best option?